content/blog/2019/root-transition/index.md

```diff
@@ -0,0 +1,150 @@
+---
+title: User guide for extenting Istio root certificate lifetime
+description: The user guide to extend Istio root certificate lifetime.
```

Comment:
Learn how to extend the lifetime of the Istio root certificate.

Reply:
Done.